lib/cog_api/fake/client.ex

```diff
@@ -84,6 +86,18 @@ defmodule CogApi.Fake.Client do
     Roles.revoke(endpoint, role, group)
   end
 
+  def rule_index(command, %Endpoint{}=endpoint) do
```

Comment:
I can see your point but I don't agree in this case.

The options you're offering are:
a. stick with a format that we've found is not correct and keep going with that
b. spend a fairly large amount of time rewriting the existing API to conform to what we want

I understand consistency and that's preferable but in this case I see a lot more benefit in using the way we want now and as we have time, we can back port the others. If we don't get to them, it's not terrible. 

I'd be fine with a conditional in the factory.

If you feel strongly, I'll follow the old style because I don't think we have the time to rewrite everything right now but I wanted to put in my reasoning.